propagators/b3/b3_propagator.go

```diff
@@ -89,22 +80,21 @@ const (
 //    x-b3-spanid: {SpanId}
 //    x-b3-sampled: {SamplingState}
 //    x-b3-flags: {DebugFlag}
-type B3 struct {
-	// InjectEncoding are the B3 encodings used when injecting trace
-	// information. If no encoding is specified (i.e. `B3Unspecified`)
-	// `B3SingleHeader` will be used as the default.
-	InjectEncoding Encoding
+// The Single Header propagator is used by default.
+func New(opts ...Option) propagation.TextMapPropagator {
```

Comment:
Should this function return a `propagation.TextMapPropagator` or `B3` type?

If we return  `propagation.TextMapPropagator` then we can also unexport `B3` type and have more flexibility in the future (e.g. pointer vs struct usage depending on benchmarks).

Reply:
+1 returning `propagation.TextMapPropagator` and unexporting `B3`.

Reply:
Unexported `B3`. Done here: 3119ced